recipes/README.md

```diff
@@ -49,16 +49,26 @@ Check out [our recipe on how to export labels](https://github.com/kili-technolog
 
 ## How to do Named-Entity Recognition with Google
 
-Example based on Enron email dataset. Its downloads the data, get the first 50 emails, pre-label them with Google NLP, push both assets and predictions to Kili and prioritize the assets.
-
-1. Create a NER project and retrieve its ID.
-
-2. Make sure that your GCP authentication is set up properly (or follow [this tutorial](https://cloud.google.com/natural-language/docs/reference/libraries)).
+Example based on Enron email dataset. Its performs the following:
+  * downloads the data.
```

Comment:
For commit names please read: https://www.notion.so/Conventional-Commits-d11e2f19f503440ebd300d1893dae2bf